### PR TITLE
tomcat --devel 9.0.0.M1

### PR DESCRIPTION
**Pull request by asaph**

### CODE REVIEW

File: Library/Formula/tomcat.rb [hunk 1]
```diff
@@ -9,6 +9,20 @@ class Tomcat < Formula
 
   option "with-fulldocs", "Install full documentation locally"
 
+  devel do
+    depends_on :java => "1.8+"
```
Comment (DomT4) [inline Library/Formula/tomcat.rb]: Can you put this below the `url`, version, etc block? Thanks!

Comment (asaph) [inline Library/Formula/tomcat.rb]: Will do @DomT4

Comment (asaph) [inline Library/Formula/tomcat.rb]: @DomT4 done!

**Comment by DomT4** [comment]
I partially rewrote your commit to reorganise things a bit more, but you still get commit credit. Just an FYI so if you look at the merge commit you don't wonder what happened :smile:.

Merged in https://github.com/Homebrew/homebrew/commit/0f37e2943bba265b42bb2087ea24b86dcc614c35. Thank you once more @asaph!

**Comment by asaph** [comment]
Thanks @DomT4 :smile:

**Pull request closed without merging**